README.md

```diff
@@ -45,9 +45,19 @@ Java and C++, with no dependencies and zero garbage collection.
 We provide a [live demo](https://demo.questdb.io/) provisioned with the latest
 QuestDB release and sample datasets:
 
-- 10 years of NYC taxi trips with 1.6 billion rows
-- live trading data from a cryptocurrency exchange
-- geolocations of 250k unique ships over time
+- Trips: 10 years of NYC taxi trips with 1.6 billion rows
+- Trades: live crytocurrency market data with 30M+ rows per month
+- Pos: geolocations of 250k unique ships over time
+
+| Query                                                                         | Execution time                                                                                                                                                                                      |
+| ----------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `SELECT sum(double) FROM trips`                                               | [0.15 secs](<https://demo.questdb.io/?query=SELECT%20sum(trip_distance)%20FROM%20trips;&executeQuery=true>)                                                                                         |
```

Comment:
It might be useful to add ";" at the end of each query? Does not read well but it is correct?

Reply:
@amyshwang yea i left it out for beauty 🙇🏼